src/components/CopyButton.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <button type="button"
+          class="button photo_copy-btn">
+    <slot v-if="!success" default />
+    <template v-if="success">Copied!</template>
+  </button>
+</template>
+
+<script>
+import Clipboard from 'clipboard';
+
+export default {
+  data: () => ({
+    clipboard: null,
+    success: false,
+  }),
+  props: {
+    toCopy: {
+      required: true,
+    },
+  },
+  mounted() {
+    this.clipboard = new Clipboard(this.$el, {
+      // eslint-disable-line no-new
```

Comment:
Curious why disable this `no-new` lint rule here. I guess it's not necessary and can be removed

Reply:
This was leftover from the previous clipboard implementation, will remove